app/experimenter/templates/base.html

```diff
@@ -99,6 +99,24 @@ <h3 class="pt-2">
       </div>
     {% endif %}
 
+    {% if messages %}
```

Comment:
Is this really the first use of `messages` in this project? I thought I had seen other messages elsewhere..

Reply:
@glasserc Could you elaborate a bit more on what you are thinking here?

Reply:
It seemed odd to me that we never used `messages` here, since we seem to have other kinds of messages already in the project. Indeed, it seems like there's a `Notification` model. Do we need both? Can we stick with the one that already exists in the project?

Reply:
I didn't use it because I didn't think this message warranted writing to the database since it's just a warning and a lack of actual action. Seems like if you use the `Notification` class you are writing the change made to the database (like X person added the datascience review) so that changes can be kind of kept track of, displayed in the review history section, and also then send that little blue message. Kinda like a changelog for changes made in the review process. I didn't think "user tried to check something they weren't allowed to check and we didn't save anything" rose to that same level of importance.. Maybe my understanding of `Notification` is off.

Reply:
To add on, I actually was intentionally not using it because I didn't want that "tried to check permission but couldnt" going into that review history section, so if I did use Notification I'd have to have some extra logic to fish it out of that section maybe...

Reply:
The module being used here, `django.contrib.messages`, is about one-off messages that happen within a request/response cycle, usually feedback about that specific response. The Notification model seems to be longer term notifications that carry a read/unread state, and is set up to tie in to sending emails (which messages wouldn't be good for).

I think that having both systems is good.

That being said, I think the styling of these messages here isn't very good. Since this is the first time we're integrating messages, I think we can make it a bit less confusing. Right now the messages blend into the navigation header too much, and it doesn't distinguish between the levels of the message (like warning, error, or info).

I'd suggest something outside of the usual flow of the page, like a toast:

![image](https://user-images.githubusercontent.com/305049/57029013-707e5d80-6bf5-11e9-943f-d2f345278306.png)

Reply:
So @mythmon, are you saying in "I think having both systems is good" that you mean you agree with the decision to use plain `messages` instead of `Notification` for this use case? 

Also, are you saying you want me to change the styling to this example styling in the PR?

Reply:
Yes, I think using `messages` in this case is the right choice.

Since the styling for messages was introduced in this PR, it's in scope to make it look not-confusing. I don't have strong opinions about how it should look, just that the current UI is confusing because it blends in with site navigation. The screenshot I posted is an idea, but it's a rough idea.

I think the best choice here would be to use Bootstrap 4.2's Toast component, but Experimenter uses Bootstrap 4.1, and I don't know how big of an upgrade that would be.

Reply:
OK, I don't mind changing the styling. Perhaps I'll run past @jaredkerim when he gets back and @shell1 to see what hopes and dreams they would have for the styling of these warning messages?

Reply:
OK, thanks for explaining. I agree that having both is the right decision.